plugins/cybereason/plugin.spec.yaml

```diff
@@ -562,3 +562,31 @@ actions:
         description: Remediate items response
         type: remediate_items
         required: true
+  remediate_items:
+    title: Remediate Items
+    description: Remediate a specific process, file or registry key if remediation is possible
+    input:
+      initiator_user_name:
+        title: Initiator User Name
+        description: Initiator user name
+        type: string
+        required: true
+        example: user@example.com
+      actions_by_machine:
+        title: Actions by Machine
+        description: Actions by machine
+        type: object
+        required: true
+        example: '{"126811122.2298225282553311122": [{"targetId": "531122333.-3391199199911692223","actionType": "KILL_PROCESS"}]}'
+      malop_id:
+        title: Malop ID
+        description: Malop ID to associate with the remediation actions
+        type: string
+        required: false
+        example: "22.2787422324806222966"
+    output:
+      response:
+        title: Malop Response
+        description: Malop response
+        type: object
```

Comment:
Should should be a custom type

Reply:
Updated to use custom type :slightly_smiling_face:

Reply:
@dmandyna-r7 it still shows object, did you forget to push a commit?